rest_api/sawtooth_rest_api/exceptions.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+from aiohttp import web
+from sawtooth_rest_api.protobuf import client_pb2 as client
+
+
+class WrongBodyType(web.HTTPBadRequest):
```

Comment:
Yes, aiohttp will take them and turn them into proper HTTP errors. They are grouped here to provide a central location for the hard-coded messages. I'm also planning ahead to when simple error messages are replaced with actual error bodies. Keeping the errors in one place will simplify the refactor.